### DISCUSSION TITLE
Module: CMSeek

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#731] [Ideas]
It might be worth it to investigate whether CMSeek would make a good BBOT module.

## Features Of CMSeek:

- Basic CMS Detection of over 180 CMS
- Drupal version detection
- Advanced Wordpress Scans
  - Detects Version
  - User Enumeration
  - Plugins Enumeration
  - Theme Enumeration
  - Detects Users (3 Detection Methods)
  - Looks for Version Vulnerabilities and much more!
- Advanced Joomla Scans
  - Version detection
  - Backup files finder
  - Admin page finder
  - Core vulnerability detection
  - Directory listing check
  - Config leak detection
  - Various other checks
- Modular bruteforce system
  - Use pre made bruteforce modules or create your own and integrate with it

**Comment by Sh4d0wHunt3rX** [comment]
Maybe WPScan is helpful too for wordpress sites.
https://github.com/wpscanteam/wpscan
